src/components/BackToTop/back-to-top.scss

```diff
@@ -3,7 +3,7 @@
   height: rem(48px);
   border-radius: rem(24px);
   background: $ibm-colors__gray-30;
-  border-color: $ibm-colors__gray-30;
+  border-color: transparent;
```

Comment:
I think this needs to be added to the `hover` block as well 

![screen shot 2019-01-28 at 12 43 15 pm](https://user-images.githubusercontent.com/11928039/51858482-72850780-22fa-11e9-8b5b-ae0779df4773.png)

Reply:
Oh good catch, I only checked in FF